examples/vnet/kubernetesvnet-azure-cni.json

```diff
@@ -0,0 +1,46 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "test",
+      "vmSize": "Standard_D2_v2",
+      "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
+      "firstConsecutiveStaticIP": "10.239.255.239",
+      "vnetCidr": "10.239.0.0/16"

```

Comment:
`vnetCidr` is the interesting bit here for a new Azure CNI custom VNET example api model